lib/fluent/plugin/out_mackerel.rb

```diff
@@ -1,4 +1,4 @@
-require 'mackerel/client'
+require 'mackerel-client'
```

Comment:
We prefer `require 'mackerel'`, although this also works.

Reply:
and maybe `require 'mackerel'` works with both mackerel-client-ruby 0.3.0 and 0.4.0

Reply:
Modified to require `mackerel`.

> and maybe require 'mackerel' works with both mackerel-client-ruby 0.3.0 and 0.4.0

Wonderful. Then, should I revert this commit? 068228ae8f307f5ecdd8820e678b3b8e8e76d7b2

Reply:
> Wonderful. Then, should I revert this commit? 068228ae8f307f5ecdd8820e678b3b8e8e76d7b2

It's better, but I realized fixing mackerel-client-ruby is more better… 🤒